src/environmentbase/scripts/region_arch_2_ami.py

```diff
@@ -111,14 +119,20 @@ def get_pv_ami(filtered_amis):
             filtered_amis = filter_amis(images)
 
         hvm_ami = get_hvm_ami(filtered_amis)
-        print json.dumps(hvm_ami, indent=4, separators=(',', ': '))
+        if hvm_ami:
+            ami_map['HVM64'][region] = hvm_ami["ImageId"]
+            print json.dumps(hvm_ami, indent=4, separators=(',', ': '))
+        else:
+            print '* No HVM hits for region'
```

Comment:
Add region info to this print message: 
from:
`print '* No HVM hits for region'`
to
`print '* No HVM hits for region', region`